eoflow/base/base_model.py

```diff
@@ -41,13 +41,43 @@ def load_latest(self, model_directory):
 
         return self.load_weights(checkpoints_path).expect_partial()
 
+    def _fit(self,
+             dataset,
+             num_epochs,
+             model_directory,
+             iterations_per_epoch,
```

Comment:
This was added in the signature of the train method [here](https://github.com/sentinel-hub/eo-flow/pull/14/files#diff-efc657321fafc2d11b0001b78e80cad7R78), and when calling `_fit` [here](https://github.com/sentinel-hub/eo-flow/pull/14/files#diff-efc657321fafc2d11b0001b78e80cad7R106). This should be enough to mimic previous behaviour, right?

Reply:
Oh, okay. Yes, that's enough to mimic previous behaviour.

Perhaps we could unify the behaviour in the future, so that both train and train_and_evaluate work in a similar way by default (i.e. by using the whole dataset as an epoch).